code/game/machinery/vending.dm

```diff
@@ -1355,8 +1355,8 @@
 	icon_state = "powermat"
 	products = list(/obj/item/weapon/cell/large = 10, /obj/item/weapon/cell/large/high = 10, /obj/item/weapon/cell/medium = 15, /obj/item/weapon/cell/medium/high = 15, /obj/item/weapon/cell/small = 20, /obj/item/weapon/cell/small/high = 20)
 	contraband = list(/obj/item/weapon/cell/large/super = 5, /obj/item/weapon/cell/medium/super = 5, /obj/item/weapon/cell/small/super = 5)
-	prices = list(/obj/item/weapon/cell/large = 500, /obj/item/weapon/cell/large/high = 700, /obj/item/weapon/cell/medium = 300, /obj/item/weapon/cell/medium/high = 400, /obj/item/weapon/cell/small = 100, /obj/item/weapon/cell/small/high = 200,
-				/obj/item/weapon/cell/large/super = 1200, /obj/item/weapon/cell/medium/super = 700, /obj/item/weapon/cell/small/super = 350)
+	prices = list(/obj/item/weapon/cell/large = 300, /obj/item/weapon/cell/large/high = 500, /obj/item/weapon/cell/medium = 200, /obj/item/weapon/cell/medium/high = 300, /obj/item/weapon/cell/small = 70 , /obj/item/weapon/cell/small/high = 150,
+				/obj/item/weapon/cell/large/super = 700, /obj/item/weapon/cell/medium/super = 400, /obj/item/weapon/cell/small/super = 250)
```

Comment:
I don't think price cuts on the "super" cells are warranted. People normally want big cells for stuff like RIGs and weapons, and it's normal for something like that to be pricey. Prices for regular trash tier cells, on the other hand, could be even lower. I'd say that 50 creds for a regular small cell, and 200 for a regular large cell is about right.

Also, move the "super" cell away from the contraband list and into the normal item list. Limit the available amount instead. You can cut the amount of cells overall too - no round would ever sell 10 tier 1 cells, and it's fine if tier 3 cells are in short supply.